lib/routes/security_pin/security_and_backup/security_and_backup_page.dart

```diff
@@ -269,6 +269,8 @@ class SecurityAndBackupPageState extends State<SecurityAndBackupPage>
             );
           } catch (e) {
             EasyLoading.dismiss();
+          } finally {
```

Comment:
This was removed because there's a delay between when backupNow action finishes and a BACKUP_REQUEST is in place.

Reply:
Understand, will remove this

Reply:
Have you encountered an issue where the spinner was being stuck?

Reply:
Yes, but I this can due to the backupserver loading as I used my own firebase setup which has proven to be fairly unstable. I will test it with our instance and report back.

Reply:
We should still understand why that happens and handle that case, either by error handling or adding timeouts to all EasyLoading.show() methods as a workaround.